docs/product-assembly_dm-on-openshift/src/main/asciidoc/imagestreams-file-install-proc.adoc

```diff
@@ -33,6 +33,9 @@ endif::DM[]
 ifdef::PAM[]
```

Comment:
I don't exactly understand what change you are suggesting. The instances of <class_name> are in code quotes, and I don't think italic is available for them. Olease clarify the suggested change with an example if possible. To simplify things, here is an original instance, please suggest changed version:

. To save the configuration of the default storage class into a YAML file, run the following command:
+
[subs="verbatim,macros"]
----
oc get storageclass <class_name> -o yaml >storage_config.yaml
----
+
Where `class-name` is the name of the default storage class.

Reply:
@mramendi, I haven't a clue how you interpreted my comment as recommending italics :) I'm saying change <class_name> to <class-name>, because you say "where class-name is the name..." and because using a hyphen is more consistent with lower-case replacables in code, and because you use underscores in this doc for actual code that are not replacable, like storage_config.yaml in this very example.

Reply:
@sterobin markup is breaking things in this comment so terribly that I stol don't understand it, I'll try to catch you on Slack.